loleaflet/src/control/Control.UIManager.js

```diff
@@ -404,6 +404,15 @@ L.Control.UIManager = L.Control.extend({
 		return $('.loleaflet-ruler').is(':visible');
 	},
 
+	// fullscreen
+
+	isFullscreen: function() {
+		if (document.fullscreenElement != null)
```

Comment:
you have to add conditions for all the browsers like done here:
https://github.com/CollaboraOnline/online/blob/465c435e6863a377611ccff2ef082b37c0967ba1/loleaflet/src/core/Util.js#L166

Reply:
I did changes, can you re-review it?

Reply:
I can't see other conditions.